app/views/users/index.html.erb

```diff
@@ -1,17 +1,19 @@
 <main>
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you fix the profile picture in the other views to make your app give a great user experience and make it professional :+1:

Reply:
Thanks